packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -452,13 +455,19 @@ export class EditManager<
 		this.localBranch.setHead(this.trunk.getHead());
 
 		for (const [sessionId, branch] of data.branches) {
-			const commit =
-				trunkRevisionCache.get(branch.base) ??
+			const commit = trunkRevisionCache.get(branch.base);
```

Comment:
Think this change is from debugging so I could set a breakpoint. Can probably go back to the way it was before